### PR TITLE
adding missing python dep for dockerfile

### PR DESCRIPTION
**Pull request by darrylmorton-digicatapult**
- [x] adding missing docker python dep
- [x] split dockerfile into build and runtime segments
- [x] members api returns correct output
- [x] depcheckrc added for pg
- [x] package.json deps cleanup
- [x] npm ci --production

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,20 +1,32 @@
...
 ENV NPM_CONFIG_LOGLEVEL ${LOGLEVEL}
+RUN apk update && \
+  apk add python make build-base && \
+  rm -rf /var/cache/apk/*
+
+WORKDIR /vitalam-identity-service
+
+# Install base dependencies
+COPY . .
+RUN npm install
```
Comment (mattdean-digicatapult) [inline Dockerfile:18]: I think this should be `npm ci --production`

File: Dockerfile [hunk 2]
```diff
@@ -1,20 +1,32 @@
...
 
 # Install base dependencies
 RUN npm -g install npm@latest
 
-WORKDIR /vitalam-identity-service
+COPY --from=build /vitalam-identity-service .
 
-COPY . .
-RUN npm install --production
+RUN npm install
```
Comment (mattdean-digicatapult) [inline Dockerfile:29]: Is there a need to `npm install` or similar here. Surely we did that already in the builder?

**Review by mattdean-digicatapult — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 3]
```diff
@@ -14,19 +14,18 @@ RUN apk update && \
 WORKDIR /vitalam-identity-service
 
 # Install base dependencies
+RUN npm -g install npm@latest
+
 COPY . .
-RUN npm install
+RUN npm ci install
```
Comment (dblane-digicatapult) [inline Dockerfile:20]: Its just `npm ci`

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 3 above)
Comment (mattdean-digicatapult) [reply Dockerfile:20]: literally

```
RUN npm ci --production
```

is what we want here I believe

File: Dockerfile (hunk 1 above)
Comment (darrylmorton-digicatapult) [reply Dockerfile]: done

**Review by mattdean-digicatapult — APPROVED** [review]

**Review by mattdean-digicatapult — CHANGES_REQUESTED** [review]
~Version bump?~

~Did this miss the check version somehow?~

**Review by mattdean-digicatapult — APPROVED** [review]
Sorry ignore my request for changes. No releases were published because of this issue

**Comment by mattdean-digicatapult** [comment]
From the helm test: 

```
Invalid environment variables:
    AUTH_ISSUER: Invalid url: "aHR0cHM6Ly92aXRhbGFtLmV1LmF1dGgwLmNvbS8="
    AUTH_JWKS_URI: Invalid url: "aHR0cHM6Ly92aXRhbGFtLmV1LmF1dGgwLmNvbS8ud2VsbC1rbm93bi9qd2tzLmpzb24="
    AUTH_TOKEN_URL: Invalid url: "aHR0cHM6Ly92aXRhbGFtLmV1LmF1dGgwLmNvbS9vYXV0aC90b2tlbg=="
 Missing environment variables:
    API_HOST: undefined
```

where has this come from? They are definitely not valid urls

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**